### PR TITLE
cppcheck performance fixes in peripherals/

### PR DESCRIPTION
**Pull request by tobbi**
Simple ones. Removing assign to a variable that we don't check

### CODE REVIEW

File: xbmc/peripherals/bus/osx/PeripheralBusUSB.cpp [hunk 1]
```diff
@@ -137,10 +137,10 @@ void CPeripheralBusUSB::DeviceDetachCallback(void *refCon, io_service_t service,
     
     CLog::Log(LOGDEBUG, "USB Device Detach:%s, %s\n",
       privateDataRef->deviceName.c_str(), privateDataRef->result.m_strLocation.c_str());
-    result = IOObjectRelease(privateDataRef->notification);
```
Comment (anaconda) [inline xbmc/peripherals/bus/osx/PeripheralBusUSB.cpp]: It seems you can get rid of it, line 124.

**Comment by tobbi** [comment]
Removed result variable.

**Comment by Jalle19** [comment]
jenkins build and merge

**Comment by Memphiz** [comment]
consider not using build and merge when jenkins is so crowded - it will fail most of the time then - jenkins build this please

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by Jalle19** [comment]
@Memphiz okay, I don't usually log into and check how queued up it is.

**Pull request merged**